src/codeu/chat/common/BasicView.java

```diff
@@ -52,4 +52,9 @@
   //   Return all messages whose id is found in the given collection.
   Collection<Message> getMessages(Collection<Uuid> ids);
 
+  // GET VERSION
+  //
+  //   Return the server's version.
```

Comment:
it's usually nice to explain that it can return null in case of an error (although i see that the other method docs aren't nice like that)